config/settings.yml

```diff
@@ -311,7 +311,6 @@ mdot:
   url: https://internal-dsva-vagov-staging-fwdproxy-1821450725.us-gov-west-1.elb.amazonaws.com:4466
   api_key: abcd1234abcd1234abcd1234abcd1234abcd1234
   mock: false
-  enabled: false
```

Comment:
This is most definitely enabled, so I assume this line is a leftover from a time long past.

Reply:
It looks like this setting is probably not used. I searched for [mdot.enabled](https://github.com/search?q=repo%3Adepartment-of-veterans-affairs%2Fvets-api%20%22mdot.enabled%22&type=code) and nothing showed up. 👍 to remove.

Reply:
Thanks for the double-check, @rmtolmach !